app/controllers/concerns/admin/list_paginated_users.rb

```diff
@@ -11,15 +11,19 @@ module Admin::ListPaginatedUsers
   RECORDS_PER_PAGE = 5
 
   private
-    def list_paginated_users(users:, template:, legacy_template:)
+    def list_paginated_users(users:, template:, legacy_template:, single_result_redirect_path: nil)
```

Comment:
Changed the `yield` to include the `pagination` as well, but thought it would be even nicer to just fully handle the concern of redirecting if there's a single result here. Lmkwyt